src/app.js

```diff
@@ -1 +1,42 @@
+/* eslint-disable no-console */
 'use strict';
+
+const { terminal } = require('./terminal');
+const { generateRandomNumber } = require('./generator');
+const { compareNumbers } = require('./compareNumbers');
+
+const randomNumber = generateRandomNumber();
+
+const playGame = () => {
+  terminal.question('Guess a number: ', (input) => {
+    const formattedInput = input.replace(/\s/g, '');
+    const isValidNumber = new Set(formattedInput).size === 4
+      && !isNaN(formattedInput);
+
+    if (!isValidNumber) {
+      console.log('Enter a number consisting of 4 different digits');
```

Comment:
Use your terminal module for all input/output. There is a `.write` method in readline ([doc](https://nodejs.org/docs/latest-v18.x/api/readline.html#rlwritedata-key)), or you can make a function that calls console.log in that module.